packages/ember-toucan-core/src/components/form/controls/character-count.ts

```diff
@@ -0,0 +1,38 @@
+import Component from '@glimmer/component';
+import { tracked } from '@glimmer/tracking';
+import { assert } from '@ember/debug';
+
+export interface ToucanFormControlCharacterCountComponentSignature {
+  Element: HTMLSpanElement;
+  Args: {
+    /**
+     * The current number of characters inside the input
+     */
+    current?: number;
+
+    /**
+     * The max amount allowed for this input
+     */
+    max: number;
+  };
+}
+
+// eslint-disable-next-line ember/no-empty-glimmer-component-classes
+export default class ToucanFormControlCharacterCount extends Component<ToucanFormControlCharacterCountComponentSignature> {
+  @tracked current = this.args.current ?? 0;
```

Comment:
You don't need to use `@tracked` for only providing a default value, as it is only derived state. So a getter should be sufficient. Also this is not event used in the template, it still has `@current`!

But personally I don't know if it's worth it to support an optional arg here, maybe just make it required as in the commit before?